frontend/src/pages/RegisterStartup.tsx

```diff
@@ -0,0 +1,317 @@
+    name: "",
+    surname: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+    company: "",
+    registercompany: false,
+    startup: false,
+    entrepreneur: false,
+    legal: false,
+  });
+
+  const [errors, setErrors] = useState<ErrorState>({});
+  const [status, setStatus] = useState<"idle" | "loading" | "success" | "error">("idle");
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    const { name, value, type, checked, files } = e.target as HTMLInputElement;
+    if (type === "checkbox") {
+      setForm({ ...form, [name]: checked });
+    } else if (type === "file" && files) {
+      setForm({ ...form, [name]: files[0] });
+    } else {
+      setForm({ ...form, [name]: value });
+    }
+  };
+
+  const validate = (): boolean => {
+    const newErrors: Record<string, string> = {};
+    if (!form.email) newErrors.email = "Не ввели електронну пошту";
+    else if (!/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(form.email)) newErrors.email = "Не вірна пошта";
+    if (!form.password) newErrors.password = "Не ввели пароль";
+    else if (form.password.length < 8) newErrors.password = "Пароль дуже малий";
+    if (form.confirmPassword !== form.password)
+      newErrors.confirmPassword = "Не ввели пароль ще раз";
+    if (!form.company) newErrors.company = "Не ввели назву компанії";
+    if (!form.name) newErrors.name = "Не ввели прізвище";
+    if (!form.surname) newErrors.surname = "Не ввели ім’я";
+    if (!form.registercompany && !form.startup) newErrors.startup = "Виберіть кого ви представляєте";
+    if (!form.entrepreneur && !form.legal) newErrors.person = "Виберіть який суб’єкт господарювання ви представляєте";
+    setErrors(newErrors);
+    return Object.keys(newErrors).length === 0;
+  };
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+    if (!validate()) return;
+    setStatus("loading");
+    try {
+      await new Promise((resolve) => setTimeout(resolve, 1500));
+      setStatus("success");
+    } catch {
+      setStatus("error");
+    }
+  };
+
+  if (status === "success") {
+    return (
+      <div className="page">
+        <header className="header">
+          <div className="logo">MyCompany</div>
+          <nav>
+            <a href="#">О нас</a>
+            <a href="#">Предприятия</a>
+          </nav>
+            <div className="search-container">
+              <input type="text" placeholder="Поиск..."/>
+              <a href="#"><span className="search-icon">🔍</span></a>
+            </div>
+          <nav>
+            <a href="#">Войти</a>
+            <a href="#">Регистрация</a>
+          </nav>
+        </header>
+
+        <div className="container">
+          <div className="success-card">
+            <h2>Registration successful!</h2>
+            <p>Check your email to confirm your account.</p>
+          </div>
+        </div>
+
+        <footer className="footer"></footer>
+      </div>
+    );
+  }
```

Comment:
The success view header shows "MyCompany" while the main header uses "CraftMerge", creating inconsistent branding after registration; use the same header/logo component in both states. Replace the success header with the same nav/logo as the main header.
```
if (status === "success") {
  return (
    <div className="page">
      <header className="navigation">{/* same nav/logo markup as below */}</header>
      ...
    </div>
  );
}
```